lib/resources/majors.tsx

```diff
@@ -5164,6 +5164,201 @@ const bsNeuro: Major = {
+      fine_requirements: [
+        {
+          required_credits: 0,
+          description:
+            'Clark Scholars may also take the following series of one-credit courses, as the equivalent of 1 three-credit course.<br />' +
+            'EN.660.106 Clark Scholars Leadership Challenge' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.660.270 Clark Scholar Engineering Design I' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.660.380 Clark Scholar Engineering Design II	',
+          criteria: 'EN.660.106[C]^OR^EN.660.270[C]^OR^EN.660.380[C]',
+        },
+        {
+          required_credits: 0,
+          description:
+            'Students may use only one Marketing and Communications course as an upper-level elective' +
+            ' for the Entrepreneurship and Management minor. <br />' +
+            'EN.660.352	New Product Development' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.660.358	International Marketing' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.660.420	Marketing Strategy' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.660.450	Advertising & Integrated Marketing Communication' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.660.453	Digital and Social Media Marketing' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.661.306	Special Topics in Professional Writing: Freelance Travel Writing' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.661.355	Special Topics in Professional Writing: Blogging about Food and Culture' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.661.360	Marketing Your Start-up' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.661.370	Storytelling with Data' +
+            '<br /> <i>OR</i> <br />' +
+            'EN.661.380	Decision Analytics',
+          criteria:
+            'EN.660.352[C]^OR^EN.660.358[C]^OR^EN.660.420[C]^OR^EN.660.450[C]^OR^EN.660.453[C]^OR^EN.661.306[C]^OR^EN.661.355[C]^OR^EN.661.360[C]^OR^EN.661.370[C]^OR^EN.661.380[C]',
+        },
+      ],
+    },
+    {
+      name: 'Entrepreneurship and Management Capstone',
+      required_credits: 3,
+      min_credits_per_course: 3,
+      description: 'Students must complete one 400-level capstone course.',
+      criteria:
+        'EN.660.410[C]^OR^EN.660.411[C]^OR^EN.660.419[C]^OR^EN.660.459[C]^OR^EN.660.460[C]^OR^EN.663.453[C]',
+      fine_requirements: [
+        {
+          required_credits: 3,
+          description:
+            '<b>EN.660.410</b> <br />Computer Science Innovation and Entrepreneurship**',
+          criteria: 'EN.660.410[C]',
+        },
+        {
+          required_credits: 3,
+          description:
+            '<b>EN.660.411</b> <br />Corporate Strategy and Business Failure**',
+          criteria: 'EN.660.411[C]',
+        },
+        {
+          required_credits: 3,
+          description: '<b>EN.660.419</b> <br />Strategy Consulting**',
+          criteria: 'EN.660.419[C]',
```

Comment:
These shouldn't be fine requirements because you only need one course out of all of these